pkgs/development/python-modules/pyjnius/default.nix

```diff
@@ -0,0 +1,33 @@
+{ buildPythonPackage
+, cython
+, fetchPypi
+, isPy3k
+, jdk
+, lib
+, six
+, pythonOlder
```

Comment:
Where are isPy3k and pythonOlder used? I believe they can be removed

Reply:
OK, I removed